### DISCUSSION TITLE
Generate A Verbatim Code Chunk or Inline R Expression in knitr Output | /en/2017/11/knitr-verbatim-code-chunk/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1462] [Comments]
When you write a code chunk or inline expression in a knitr source document (e.g., R Markdown), and it is parsed and evaluated, you won&rsquo;t be surprised. That is what knitr is supposed to do. Then &hellip;

https://yihui.org/en/2017/11/knitr-verbatim-code-chunk/

**Comment by JMLuther** [comment]
thanks, very helpful. 
The Rmd text editor in RStudio does not handle this new ````{verbatim} option very well- it works (eg the html output is perfect), but the document formatting is messed up; the inline code highlighting (eg mine in dark mode uses white to indicate text, yellow for code) and the Rmd outline pane is messed up (would add a screenshot but cannot here).
(R 4.2.1, RStudio 2022.12.0 build 353, on windows 10; knitr 1.4.1)

**Reply by yihui** [reply]
I never use dark themes so I haven't noticed this before. Just tried a dark theme and you are right:

![image](https://user-images.githubusercontent.com/163582/210631006-4ec0ee64-4f92-4425-9573-5af5356074b6.png)

This is an RStudio IDE problem. You may file an issue there: https://github.com/rstudio/rstudio/issues

Alternatively, you can use the visual editor: https://rstudio.github.io/visual-markdown-editing/

**Reply by JMLuther** [reply]
I think it also causes an issue in other themes, but I'm so accustomed to seeing the text/code formatting using my preferences I can't interpret the others very well.

I'm filing an issue at RStudio, thanks!

**Comment by JMLuther** [comment]
I filed [an issue here](https://github.com/rstudio/rstudio/issues/12527)
The example text does not come across clearly because it apparently interprets the {verbatim} text block out of the example.

**Reply by yihui** [reply]
> The example text does not come across clearly

Use five backticks: https://yihui.org/en/2018/11/hard-markdown/#1-how-to-write-verbatim-content-especially-text-that-contains-three-backticks

It's understandably hard :)
